functions/AFQ_Create.m

```diff
@@ -188,8 +193,25 @@
 afq.params.track.seedVoxelOffsets = [0.25 0.75];
 % Mask from which to initialize tracking
 afq.params.track.faMaskThresh = 0.30;
+
+% Parameters relevant to mrTrix.
+% Beware, the code is being maintained for both mrTrix2 and mrTrix3. 
+% Consider that mrTrix2 is called obsolete by the developers, with no updates 
+% http://community.mrtrix.org/t/mrtrix-tutorial-error/141
+% Function names change, and there are many new options in mrTrix3.
 % Number of fibers to track. This parameter is only relevant for mrTrix
 afq.params.track.nfibers = 500000;
+% Choose algorithm for tracking with mrTrix
+% Options if you have version 2:
+%    'probabilistic tractography': 'SD_PROB'
+%    'deterministic tractogrpahy based on spherical deconvolution': 'SD_STREAM'
+%    'deterministic tractogrpahy based on a tensor model': 'DT_STREAM'
+% Options if you have version 3:
+%     FACT, iFOD1, iFOD2, Nulldist1, Nulldist2, SD_Stream, 
+%                         Seedtest, Tensor_Det, Tensor_Prob (default: iFOD2).
+afq.params.track.mrTrixAlgo = 'iFOD2';
+% Specify here if you want multishell true or false
+afq.params.track.multishell = true;
```

Comment:
This should default to false, or later code should be changed to not break other AFQ pipelines.